flycheck-dmd-dub.el

```diff
@@ -383,14 +389,7 @@ to `fldd--cache-file' to reuse the result of dub describe."
 
 (defun fldd--dub-describe-cache-file-name ()
   "The file name to cache the describe output for PROJECT-DIR."
-  (expand-file-name
-   "dub_describe.json"
-   (expand-file-name
-    (let ((project-dir (fldd--get-project-dir)))
-      (if (eq system-type 'windows-nt)
-          (replace-regexp-in-string ":" "_" project-dir)
-        project-dir))
-    fldd--cache-dir)))
+  (concat fldd--cache-dir (fldd--get-project-dir) "dub_describe.json"))
```

Comment:
Shouldn't one use a some kind of join path function here instead of concat?

Reply:
https://stackoverflow.com/questions/3964715/what-is-the-correct-way-to-join-multiple-path-components-into-a-single-complete